README.md

```diff
@@ -25,14 +25,21 @@ In this challenge, create a web API around the following resources: `Projects` a
 Demonstrate your understanding of this Sprint's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead.
 
 - [ ] Mention two parts of Express that you learned about this week.
+      Middleware & Routes
 
 - [ ] Describe Middleware?
+      Middleware is a function thats sits in the middle between a request and the final intended destination.
 
 - [ ] Describe a Resource?
+      Database?
```

Comment:
grouping / category of data used in the app. (users / projects / video games) essentially if you think about an excel sheet what would make sense to be saved here